databox-build-core

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m' 
+NC='\033[0m'
```

Comment:
could maybe wrap these up as `green`, `red` shell functions instead, to avoid need to remember to close the escape

Reply:
done